### ISSUE TITLE
Missing I2C interface for Inky pHAT

### ISSUE THREAD

**Issue by danieldulaney** [pinout-xyz/Pinout.xyz#387]
In addition to the documented SPI interface, the Inky pHAT (also most likely the wHat, but I don't have access to that hardware) also uses the I2C interface pins (in their normal configuration) for device discovery.

I haven't been able to find any documentation on the interface except the board autodiscovery code in the phat Python library (https://github.com/pimoroni/inky/blob/f5b5bb557b0ab6770b732b23ef4898e992bd0ca3/library/inky/eeprom.py#L55-L61).

Would it make sense to add the I2C pins for these boards?

**Comment by Gadgetoid** [comment]
Yes, it would. Actually the Inky pHAT page is hideously out of date and needs some TLC- points at the wrong library and doesn't achknowledge newer boards. Working on it alongside getting Impression added.
